db/migrations/20181221164327-create-deleted-subscription.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('DeletedSubscriptions', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.BIGINT,
+    },
+    githubId: {
+      allowNull: false,
+      type: Sequelize.BIGINT,
+    },
+    channelId: {
+      allowNull: false,
+      type: Sequelize.STRING,
+    },
+    type: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      defaultValue: 'repo',
+    },
+    slackWorkspaceId: {
+      type: Sequelize.BIGINT,
+      allowNull: false,
+    },
+    installationId: {
+      type: Sequelize.BIGINT,
+      allowNull: true,
+      hooks: true,
+    },
+    creatorId: {
+      type: Sequelize.BIGINT,
+      allowNull: true,
+    },
+    settings: {
+      type: Sequelize.JSON,
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE,
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE,
+    },
```

Comment:
do we want a `deletedAt` column? (assuming that the values from `createdAt` and `updatedAt` are copied directly from the real subscription)

Reply:
Oh, that's a good idea. I'll add that.

Reply:
Done :)